src/twisted/internet/interfaces.py

```diff
@@ -1508,15 +1517,15 @@ def removeSystemEventTrigger(triggerID):
             C{triggerID}.
         """
 
-    def callWhenRunning(callable, *args, **kw):
+    def callWhenRunning(f: Callable[..., Any], *args, **kw):
```

Comment:
This is still an interface change callable -> f

Reply:
I took this out